proto/viam/app/v1/app.proto

```diff
@@ -411,6 +423,51 @@ message DeleteOrganizationMemberRequest {
 
 message DeleteOrganizationMemberResponse {}
 
+// Third Party Org Services
+
+message BillingAddress {
+  string address_line_1 = 1;
+  optional string address_line_2 = 2;
+  string city = 3;
+  string state = 4;
+  string zipcode = 5;
+}
+
+message EnableBillingServiceRequest {
+  string org_id = 1;
+  BillingAddress billing_address = 2;
+}
+
+message OrganizationSetSupportEmailRequest {
+  string org_id = 1;
+  string email = 2;
+}
+message OrganizationSetSupportEmailResponse {}
+
+message OrganizationGetSupportEmailRequest {
+  string org_id = 1;
+}
+
+message OrganizationGetSupportEmailResponse {
+  string email = 1;
+}
+
+message EnableBillingServiceResponse {}
```

Comment:
sorry this is soooo nitty but can we move this up so its paired with its request? all the others are paired